mock/instance.go

```diff
@@ -7,3 +7,11 @@ import (
 func (m *Mock) CountInsertedObservations(ctx context.Context, instanceID string) (count int64, err error) {
 	return 0, m.checkForErrors()
 }
+
```

Comment:
☝️ If this method has been removed from the interface can this also be removed if its not needed anymore?